docs/source/host_challenge.md

```diff
@@ -149,7 +149,7 @@ Open [`challenge_config.yml`](https://github.com/Cloud-CV/EvalAI-Starters/blob/m
 
 Please ensure the following fields are set to the following values for code-upload based challenges:
```

Comment:
I think we should remove all of these, right?

Reply:
This is removed in a follow-up PR.